crates/core/src/db/relational_db.rs

```diff
@@ -59,6 +63,7 @@ pub struct RelationalDB {
 
     // Release file lock last when dropping.
     _lock: Arc<File>,
+    pub(crate) config: Arc<RwLock<DatabaseConfig>>,
```

Comment:
I think we should probably keep this private.

Reply:
Ok, I made this private and created a helper method, it is useful because it abstracts the locking...